appcues/src/main/java/com/appcues/debugger/DebuggerViewModel.kt

```diff
@@ -38,9 +40,9 @@ internal class DebuggerViewModel(
 
     sealed class UIState {
         object Creating : UIState()
-        object Idle : UIState()
+        data class Idle(val mode: DebugMode) : UIState()
         data class Dragging(val dragAmount: Offset) : UIState()
-        data class Expanded(val deepLinkPath: String? = null) : UIState()
+        data class Expanded(val mode: DebugMode) : UIState()
```

Comment:
Imagining that the new capture confirmation dialog may extend from `Expanded` when the mode is `ScreenCapture`, but I'm not totally sure yet - might just be a modal popup on top, haven't gone down that path too far yet.

Reply:
sure thats fine, I think it would be very slick if we reuse all the animation work we have and just change the content inside the expanded area instead of a totally different dialog popping up, good to keep options open if we want to propose a new UI